SMLHelper/Assets/FishPrefab.cs

```diff
@@ -69,7 +69,7 @@ public sealed override GameObject GetGameObject()
             forces.handleGravity = true;
             forces.underwaterDrag = 1f;
             forces.underwaterGravity = 0;
-            forces.waterDepth = Ocean.main.GetOceanLevel();
+            forces.waterDepth = Ocean.GetOceanLevel();
```

Comment:
**TODO** This will be updated to compile differently for the two [build configurations](https://docs.microsoft.com/en-us/visualstudio/ide/understanding-build-configurations?view=vs-2019) using [preprocessor directives](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives/)
```CS
#if SUBNAUTICA 
forces.waterDepth = Ocean.main.GetOceanLevel();
#elif BELOWZERO
forces.waterDepth = Ocean.GetOceanLevel();
#endif
```